### ISSUE TITLE
Error 1015 

### ISSUE THREAD

**Issue by Per0xide0** [Csardelacal/YCH-Issues#90]
When I click on an image in an auction
 to view it in the same tab I am met with an error screen that claims I have been rate limited and banned from accessing the site, however if I open the image in a new tab it loads fine, I have had this error of both pc and mobile

![Screenshot_20230727-230227_Chrome](https://github.com/Csardelacal/YCH-Issues/assets/140777283/5d0456cc-975b-4953-99b5-4446007b7487)

**Comment by Csardelacal** [comment]
Hey there! The error is a very unfortunate wording from Cloudflare (which provides us with CDN services).

A bit of background: The old mechanism for storing images is becoming very overwhelmed with the traffic from crawlers and stuff. So we asked Cloudflare to send a 429 code to crawlers that request more than one image per second. Unfortunately Cloudflare also adds this message about being banned. Which is not true and will disappear after a few seconds.

The new storage service will not use this rate limiting mechanism and therefore should no longer print this message. Until the migration is complete, that's the unfortunate reality of it. ^^;

TL;DR: You're not actually banned. Reloading after a few seconds disperses the error. The fix is being rolled out but may take a while to be actually available.
